stubs/croniter/@tests/stubtest_allowlist.txt

```diff
@@ -2,9 +2,4 @@
 croniter.croniter.EXPANDERS
 croniter.croniter.HashExpander
 croniter.croniter.VALID_LEN_EXPRESSION
-croniter.croniter.hash_expression_re
-croniter.croniter.only_int_re
-croniter.croniter.special_weekday_re
-croniter.croniter.star_or_int_re
-croniter.croniter.step_search_re
```

Comment:
I'm pretty confused about why stubtest errors on these being missing from the stub when run on Python 3.9, but not when run on Python 3.10. I can reproduce this behaviour locally (the same version of `croniter` is installed, `1.3.8`).

Here's the results on Python 3.9 if I remove these items from the allowlist:

```
error: croniter.croniter.hash_expression_re is not present in stub
Stub: in file C:\Users\alexw\coding\typeshed\stubs\croniter\croniter\croniter.pyi
MISSING
Runtime:
re.compile('^(?P<hash_type>h|r)(\\((?P<range_begin>\\d+)-(?P<range_end>\\d+)\\))?(\\/(?P<divisor>...

error: croniter.croniter.only_int_re is not present in stub
Stub: in file C:\Users\alexw\coding\typeshed\stubs\croniter\croniter\croniter.pyi
MISSING
Runtime:
re.compile('^\\d+$')

error: croniter.croniter.special_weekday_re is not present in stub
Stub: in file C:\Users\alexw\coding\typeshed\stubs\croniter\croniter\croniter.pyi
MISSING
Runtime:
re.compile('^(\\w+)#(\\d+)|l(\\d+)$')

error: croniter.croniter.star_or_int_re is not present in stub
Stub: in file C:\Users\alexw\coding\typeshed\stubs\croniter\croniter\croniter.pyi
MISSING
Runtime:
re.compile('^(\\d+|\\*)$')

error: croniter.croniter.step_search_re is not present in stub
Stub: in file C:\Users\alexw\coding\typeshed\stubs\croniter\croniter\croniter.pyi
MISSING
Runtime:
re.compile('^([^-]+)-([^-/]+)(/(\\d+))?$')
```

But I can't access these variables at runtime, not on Python 3.9 or Python 3.10:

```pycon
>>> import croniter.croniter
>>> croniter.croniter.hash_expression_re
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'croniter' has no attribute 'hash_expression_re'
```

So why does stubtest error on these being missing on the stub when run on Python 3.9? And what's different about Python 3.10?

Reply:
I don't know what's different about 3.10 but you can't access them because `croniter.croniter` is a type that shadows the `croniter.croniter` module. I can access these regexes in 3.9:
```
In [19]: sys.modules["croniter.croniter"].step_search_re
Out[19]: re.compile(r'^([^-]+)-([^-/]+)(/(.*))?$', re.UNICODE)
```

Reply:
> I don't know what's different about 3.10 but you can't access them because `croniter.croniter` is a type that shadows the `croniter.croniter` module.

Yeah, I get that -- the behaviour on 3.10 makes more sense to me! But I don't get what changed. I can access the regexes through the same `sys.modules` indirection on 3.10 as well:

```pycon
Python 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import croniter.croniter
>>> import sys
>>> sys.modules["croniter.croniter"].step_search_re
re.compile('^([^-]+)-([^-/]+)(/(\\d+))?$')
```